definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2770,19 +2770,7 @@
     "ShowSummary": "Y",
     "SecurityClassification": "Public"
   },
-  {
-    "LiveFrom": "23/07/2020",
-    "CaseTypeID": "DIVORCE",
-    "ID": "serviceApplicationReceived",
-    "Name": "Service application received",
-    "Description": "Service application received",
-    "DisplayOrder": 11,
-    "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AosAwaiting;AosDrafted",
```

Comment:
this was moved to definitions/divorce/json/CaseEvent/CaseEvent-resp-journey-roles-and-permissions-nonprod.json